templates/base/src/utils/Logger/logger.js

```diff
@@ -0,0 +1,125 @@
+
+import fetch from 'isomorphic-unfetch';
```

Comment:
In Next.js version 9.4.1 we don't have to use 'isomorphic-unfetch' lets upgrade and clean this up in next PR.

Created story for it.